packages/core/src/index.html

```diff
@@ -85,7 +85,10 @@
                 var antiClickjack = document.getElementById('antiClickjack');
                 antiClickjack.parentNode.removeChild(antiClickjack);
             } else {
-                top.location = self.location;
+                const isAuthFlow = /\/(silent-callback|front-channel)($|\/)/.test(self.location.pathname);
+                if (!isAuthFlow) {
+                    top.location = self.location;
+                }
```

Comment:
This prevents the `silent-callback.html` and `front-channel.html` iframes to be navigated in the main window